suites/e2e/tests/always-fail/index.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
     {
       title: 'Kill the device under test',
       run: async function (test) {
+        // This function doesn't exist anywhere and should be failing the test every time.
```

Comment:
+ 1 for this comment - maybe amend the comment to explain why it is here, if we expect it to fail every time. I remember i causing some confusion with some others who were using leviathan for the first time

Reply:
Yeah, I was just adding a small comment on this since I missed it again. But, the file's documentation has it detailed out quite nicely. Check the full test file to see it. Let me know if i should add more.